src/governance.ts

```diff
@@ -302,6 +305,31 @@ export class GovernanceCanister {
     return undefined;
   };
 
+  /**
+   * Refreshes neuron and returns neuronId when successful
+   * Uses query call only.
+   *
+   * @throws {@link UnrecognizedTypeError}
+   */
+  public claimOrRefreshNeuron = async (
+    request: ClaimOrRefreshNeuronRequest
+  ): Promise<NeuronId | undefined> => {
+    const rawRequest = fromClaimOrRefreshNeuronRequest(request);
+    const rawResponse = await this.service.manage_neuron(rawRequest);
+    const { command } = rawResponse;
+    if (command.length && "ClaimOrRefresh" in command[0]) {
```

Comment:
funky alternative: `"ClaimOrRefresh" in (command?.[0] || {}))`

Reply:
TS didn't like it...